js/utils/selectors.js

```diff
@@ -0,0 +1,17 @@
+import $ from 'jquery';
```

Comment:
Created this module as a place to centralize and cache global selectors (i.e. selectors that live outside any particular view). I took the approach of making them methods that will do the initial DOM query the first time they are called, which means if for some reason someone follows a code path where they are never needed, we'll save ourself an unnecessary DOM query.